src/Appwrite/Auth/Auth.php

```diff
@@ -67,7 +67,7 @@ class Auth
     public const TOKEN_EXPIRATION_LOGIN_LONG = 31536000;      /* 1 year */
     public const TOKEN_EXPIRATION_LOGIN_SHORT = 3600;         /* 1 hour */
     public const TOKEN_EXPIRATION_RECOVERY = 3600;            /* 1 hour */
-    public const TOKEN_EXPIRATION_CONFIRM = 3600 * 24 * 7;    /* 7 days */
+    public const TOKEN_EXPIRATION_CONFIRM = 3600 * 1;         /* 1 hour */
```

Comment:
Is this only used for magic URLs or in other places as well ?

Reply:
Its in email verification as well, password reset too I would guess.
But all of that si fine. If you get email token, there is no reason for it to be valid for 7 days. 1 hour is more than enough.